src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CompleteClarifyingQuestionsHandler.java

```diff
@@ -66,18 +69,35 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
             Direction direction = clarifyingQuestionsDirectionOptional.get().getValue();
             List<IdValue<ClarifyingQuestion>> questions = direction.getClarifyingQuestions();
 
-            List<IdValue<ClarifyingQuestionAnswer>> answers = questions
-                .stream().map(x -> mapDefaultAnswer(x, direction, callback.getEvent()))
-                .collect(Collectors.toList());
-
             Optional<List<IdValue<ClarifyingQuestionAnswer>>> existingAnswers = asylumCase.read(CLARIFYING_QUESTIONS_ANSWERS);
+
+            // set default answer for unanswered clarifying questions
+            List<IdValue<ClarifyingQuestionAnswer>> answers = questions.stream()
```

Comment:
Makes sense to rename 'answers' to 'defaultAnswers' or something like this.

Reply:
default answers are added in `mapDefaultAnswer` method. This variable will have all answers i.e, answers submitted by appellant and the default answers for pending questions